flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -121,4 +135,36 @@ protected Catalog createCatalog(String name, Map<String, String> properties, Con
   public static Configuration clusterHadoopConf() {
     return HadoopUtils.getHadoopConfiguration(GlobalConfiguration.loadConfiguration());
   }
+
+  private void loadHiveConf(Configuration configuration, Map<String, String> properties) {
+    String hiveConfPath = properties.get(HIVE_SITE_PATH);
+    Path path = new Path(hiveConfPath);
+    if (hiveConfPath.startsWith("hdfs")) {
```

Comment:
It seems weird to limit this to HDFS and not other hadoop compatible file systems, unless I'm missing something?

Reply:
yes ,you are right ,we should support more storage to store hive configuration file

Reply:
Could we also use a named constant for these string literals for the filesystem scheme? Maybe HDFS_SCHEME in this case would work.